### PR TITLE
Add cloudbuild rule to send screenshot email notification

### PR DESCRIPTION
**Pull request by shifucun**
On each commit of a PR, the screenshot is triggered. Once the screenshot test is complete, author will receive an email with the screenshot link to verify.

### CODE REVIEW

**Comment by shifucun** [comment]
@wh1210 Please take a look

File: build-notifier/send_email.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import argparse
+import smtplib
+from google.cloud import secretmanager
+
+
+arg_parser = argparse.ArgumentParser()
+arg_parser.add_argument(
+  '--email', help='Receiver email', type=str, required=True)
```
Comment (wh1210) [inline build-notifier/send_email.py:8]: I am curious why this can fetch the email directly.

Comment (shifucun) [reply build-notifier/send_email.py:8]: git clone and git log should show the author email as it worked locally. I guess the github action has some weird permission setup.

Comment (shifucun) [reply build-notifier/send_email.py:8]: Note the email is fetched here: https://github.com/datacommonsorg/website/pull/290/files#diff-83d8e1ea99371365d0e6f1fe066d4645R54-R57

File: build-notifier/send_email.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+arg_parser.add_argument(
+  '--short_sha', help='Commit short sha', type=str, required=True)
+arg_parser.add_argument(
+  '--pull_request', help='Id of pull request', type=str, required=True)
+args = vars(arg_parser.parse_args())
+
+
+port = 587  # For SSL
+sender_email = 'datcom.ci@gmail.com'
```
Comment (beets) [inline build-notifier/send_email.py:17]: Should we use a google email address here (or we have datacommons@gmail.com)?

File: run_test.sh [hunk 3]
```diff
@@ -82,18 +98,19 @@ function run_all_tests {
 }
```
Comment (beets) [inline run_test.sh:98]: should we add the screenshot tests here? it'll be nice to have one command to use for verification prior to sending a pr

File: server/webdriver_tests/screenshot/screenshot_test.py [hunk 4]
```diff
@@ -14,11 +14,12 @@
 
 
 import time
-from base_test import WebdriverBaseTest
+from webdriver_tests.base_test import WebdriverBaseTest
 
+WIDTH = 1280
```
Comment (beets) [inline server/webdriver_tests/screenshot/screenshot_test.py:19]: feature request: we should run these tests at a few widths, at least 2 so we capture mobile browsers

File: build-notifier/send_email.py (hunk 2 above)
Comment (shifucun) [reply build-notifier/send_email.py]: done. using datacommonsorg@gmail.com

File: run_test.sh (hunk 3 above)
Comment (shifucun) [reply run_test.sh:99]: done

File: server/webdriver_tests/screenshot/screenshot_test.py (hunk 4 above)
Comment (shifucun) [reply server/webdriver_tests/screenshot/screenshot_test.py:20]: added a todo

**Pull request merged**